src/models/fhir/recognised.ts

```diff
@@ -31,7 +31,7 @@ export const hasRecognisedFields = (type: Type, parsedBundle: ParsedBundle) => {
   ) {
     throw new CodedError(
       "INVALID_DOCUMENT",
-      `Submitted HealthCert is invalid, the patient NRIC-FIN value in fhirBundle has invalid checksum. For more info, refer to the mapping table here: https://github.com/Open-Attestation/schemata/pull/38`
+      `Submitted HealthCert is invalid, the patient NRIC-FIN value in fhirBundle has an invalid checksum. For more info, refer to the documentation here: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/wiki`
```

Comment:
For invalid checksum errors, do we still direct them to the documentation? Because we didn't mention anything about NRIC-FIN there.

Reply:
actually, no need. let me remove that `for more info...`